builtin/logical/mssql/path_creds_create.go

```diff
@@ -95,7 +95,7 @@ func (b *backend) pathCredsCreateRead(ctx context.Context, req *logical.Request,
 			"name":     username,
 			"password": password,
 		}
-		if err := dbtxn.ExecuteTxQuery(ctx, tx, m, query); err != nil {
+		if err := dbtxn.ExecuteTxQueryDirect(ctx, tx, m, query); err != nil {
```

Comment:
The database secret engines under `builtin/logical/` that's not `database/` such as this one (aka "Standalone DB Engines") are deprecated and will be [removed in the near future](https://www.vaultproject.io/docs/deprecation). Doesn't hurt to update them, but note that we will be dropping them soon.

Reply:
I was just changing all uses of the existing functions in `vault` :)